Gemfile

```diff
@@ -58,6 +57,7 @@ end
 
 group :development, :test do
   gem 'rspec-rails', '~> 5.1'
+  gem 'dotenv-rails', '~> 2.7'
```

Comment:
Rubocop is complaining about the alphabetical order of the gems:
```
> Gemfile:60:3: C: [Correctable] Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem dotenv-rails should appear before rspec-rails.
  gem 'dotenv-rails', '~> 2.7'
```

I hate this rule, it always catches me out 😆